src/views/GovernanceTaskList/SideNavActions/index.scss

```diff
@@ -0,0 +1,23 @@
+.sidenav-actions {
+  border-top: 4px solid #009BC3;
+  padding-top: 12px;
+  display: flex;
+  flex-direction: column;
+  &::before{
+  }
+
+  .usa-button--unstyled {
```

Comment:
Given the way we're importing our css, this style has the ability to impact other unstyled buttons on the page, and not just the ones that exist in the left-hand nav. Can we look into using some of the uswds utility classes instead of restyling the button? Docs here: https://designsystem.digital.gov/utilities/margin-and-padding/